### PR TITLE
Add panelist api

### PR DESCRIPTION
**Pull request by chirag200666**

### CODE REVIEW

**Comment by chirag200666** [comment]
@prschmid can you please have a look at this

**Comment by chirag200666** [comment]
@prschmid when is 1.1.6 expected to be released?
Also any review comments for the PR?

**Comment by prschmid** [comment]
@chirag200666 Thank you for your patience! I'm in the process of getting things for 1.1.5 merged so that we can make that release this week. I hope to start to get to things in 1.1.6 starting next week.

As for the delay, I have a full time job at a (very) early stage startup, so things are a bit busy on my end. That being said, I very much appreciate your work and effort to make this library better and will get to it as soon as I can.

File: tests/zoomus/components/webinar/test_list_panelists.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    suite.addTest(unittest.makeSuite(ListPanelistsV2TestCase))
+    return suite
+
+
+class ListPanelistsV2TestCase(unittest.TestCase):
+    def setUp(self):
+        self.component = components.webinar.WebinarComponentV2(
+            base_uri="http://foo.com", config={"api_key": "KEY", "api_secret": "SECRET"}
+        )
+
```
Comment (prschmid) [inline tests/zoomus/components/webinar/test_list_panelists.py:20]: @chirag200666 Looks like you are missing tests here. Can I ask you to add some in? Thanks!

File: zoomus/__init__.py [hunk 2]
```diff
@@ -7,4 +7,4 @@
 
 
 __all__ = ["API_VERSION_1", "API_VERSION_2", "ZoomClient"]
-__version__ = "1.1.3"
+__version__ = "1.1.4"
```
Comment (prschmid) [inline zoomus/__init__.py:10]: Can I ask you revert this change -- I update the versions when I make the official releases (i.e. this will be 1.1.6 when I get to it).

File: tests/zoomus/components/webinar/test_add_panelists.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                "version": util.API_VERSION_2,
+            },
+        )
+
+   @responses.activate
+    def test_can_add_panelists(self):
+        responses.add(
+            responses.POST, "http://foo.com/webinar/ID/panelists",
+        )
+        response = self.component.add_panelists(panelists=[{"name": "Mary", "email": "maryjkdfdsgfshdgf@jdfdkjdglfk.jkfgdj"}])
```
Comment (prschmid) [inline tests/zoomus/components/webinar/test_add_panelists.py:31]: It's perfectly fine to have a fake email, but can we make it something like 'test@test.com' just to make it look a bit cleaner? Thanks!

File: tests/zoomus/components/webinar/test_add_panelists.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        self.assertEqual(
+            response.request.body, '{panelists: [{"name": "Mary", "email": "maryjkdfdsgfshdgf@jdfdkjdglfk.jkfgdj"}]}'
+        )
```
Comment (prschmid) [inline tests/zoomus/components/webinar/test_add_panelists.py:34]: These lines are a bit too long -- if you take a look at the rest of the codebase, we try to keep lines to about 80 characters. Can I ask you to split this on to multiple lines so it meets that standard? Thanks!

File: tests/zoomus/components/webinar/test_delete_panelists.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        )
+
+    @responses.activate
+    def test_can_delete_panelists(self):
+       responses.add(
+            responses.POST, "http://foo.com/webinar/ID/panelists",
+        )
+        response = self.component.delete_panelists(panelists=[{"name": "Mary", "email": "maryjkdfdsgfshdgf@jdfdkjdglfk.jkfgdj"}])
+        self.assertEqual(
+            response.request.body, '{panelists: [{"name": "Mary", "email": "maryjkdfdsgfshdgf@jdfdkjdglfk.jkfgdj"}]}'
```
Comment (prschmid) [inline tests/zoomus/components/webinar/test_delete_panelists.py:31]: See comments from above

**Review by prschmid — CHANGES_REQUESTED** [review]
Thank you for your effort to add this functionality in! I've requested a couple minor changes and the addition of another test. Looking forward to the next round.

**Comment by prschmid** [comment]
@chirag200666 Looks like there is a conflict due to some formatting updates that were made when upgrading black to the latest version. Can I trouble you to pull the most recent `develop` and address the issues? Thank you!

File: tests/zoomus/components/webinar/test_add_panelists.py (hunk 4 above)
Comment (chirag200666) [reply tests/zoomus/components/webinar/test_add_panelists.py:38]: reduced to 86, more than this would make it hard to read I think.

File: tests/zoomus/components/webinar/test_list_panelists.py (hunk 1 above)
Comment (chirag200666) [reply tests/zoomus/components/webinar/test_list_panelists.py:20]: added

File: zoomus/__init__.py (hunk 2 above)
Comment (chirag200666) [reply zoomus/__init__.py]: fixed.

File: tests/zoomus/components/webinar/test_add_panelists.py (hunk 3 above)
Comment (chirag200666) [reply tests/zoomus/components/webinar/test_add_panelists.py]: done.

File: tests/zoomus/components/webinar/test_delete_panelists.py (hunk 5 above)
Comment (chirag200666) [reply tests/zoomus/components/webinar/test_delete_panelists.py]: done.

**Comment by chirag200666** [comment]
@prschmid worked on all the review comments & fixed them, can you please have another look.
thanks.

File: tests/zoomus/components/webinar/test_add_panelists.py (hunk 4 above)
Comment (prschmid) [reply tests/zoomus/components/webinar/test_add_panelists.py:38]: Great thanks! As long as you ran the `black` code formatter and it didn't complain, that works for me... because the build will fail if the formatter doesn't approve. =)

**Comment by prschmid** [comment]
Thanks for all the fixes! I'll try to get to it soon.

**Review by prschmid — APPROVED** [review]
LGTM! Thank you for adding this functionality! Much appreciated!
